src/utilities/data-model.js

```diff
@@ -0,0 +1,43 @@
+import Firebase from 'firebase';
+import _ from 'lodash';
+import X2JS from 'x2js';
+import FairLandXML from '../gpx/fairland';
+
+const ROOT_URL = 'https://shining-fire-7029.firebaseio.com';
+
+export default {
+
+  fetchMapTrail(){
+
+    //Temporary until we get data from firebase containing trails
+    const x2js = new X2JS();
```

Comment:
Using fake data for now since we don't have any of this stored in Firebase - I don't believe...